eeauditor/auditors/aws/Amazon_SQS_Auditor.py

```diff
@@ -25,11 +26,19 @@
 cloudwatch = boto3.client("cloudwatch")
 
 
+def list_queues(cache):
```

Comment:
There is an error with the Boto3 list_queues call, it does not return an empty list if there are not any queues in the region (I opened an issue here about it).

You can insert something like the following within the `cache` function to get around it for the time being, as catching it with an `Exception` does not work either.

```python
listqueues = sqs.list_queues(MaxResults=1000)
if 'QueueUrls' in listqueues:
    for url in listqueues['QueueUrls']:
        ### do stuff ###
else:
    pass
```

Reply:
I've added in the suggested handling in all of the SQS checks, which will only progress further if `QueueUrls ` is in the response.   I've also added a unit test to that effect.